### ISSUE TITLE
Ollie child theme "broken" when using Create Block Theme plugin

### ISSUE THREAD

**Issue by gregfarries** [OllieWP/ollie#31]
child theme with Ollie... I've attempted to use the Create Block Theme WP plugin to create a child theme of Ollie, and it doesn't seem to work.

<img width="667" alt="Screenshot 2023-05-12 at 4 30 24 PM" src="https://github.com/OllieWP/ollie/assets/5964105/464e858e-5918-4c9a-8a7d-1916b930aa75">

I get an error stating the parent theme, Ollie isn't installed.  

I'm using the "Create child of Ollie" functionality in Create Block Theme, and this functionality works when making a child theme of say, Frostwp...

Here is the exported folder structure when creating a child of Frost:

<img width="298" alt="Screenshot 2023-05-12 at 4 33 48 PM" src="https://github.com/OllieWP/ollie/assets/5964105/7847cd08-bc81-413d-bb04-f9efe36e6f05">

But this is what happens when creating a child of Ollie:

<img width="319" alt="Screenshot 2023-05-12 at 4 34 51 PM" src="https://github.com/OllieWP/ollie/assets/5964105/a753dc54-78d2-4129-8b5d-9200a07d0efd">

Any ideas?

Thanks!

**Comment by mikemcalister** [comment]
Hey there, I'm not seeing any problem with generating a child theme with Create Block Theme. I'm able to create and activate a child theme with no error message. 

A note about the folder structure. That isn't always going to be the same theme-to-theme or site-to-site, so that doesn't necessarily indicate it's broken. If you edit the theme via the editor (wherein it saves template or template part variations to the db) that can affect the child theme it generates. 

Here's the Ollie child theme I generated: 

<img width="310" alt="Screen Shot 2023-05-13 at 12 30 38 PM" src="https://github.com/OllieWP/ollie/assets/1415737/61ec5c70-52e1-47d4-9466-3013dcd71ea4">

Because I modified my header template part, you see that was included in the child theme.

Here's the Frost child theme I generated:

<img width="310" alt="Screen Shot 2023-05-13 at 12 27 01 PM" src="https://github.com/OllieWP/ollie/assets/1415737/c642a143-d678-4592-934f-d485968e3091">

I didn't modify this theme at all, so it's a standard starter child theme with the readme, a stylesheet, and a theme.json file.

In terms of the error that is showing, I can't say for sure what that's about because I can't reproduce it. I guess make sure that this line in the Ollie child theme says `ollie`, just as seen here. 

<img width="414" alt="Screen Shot 2023-05-13 at 12 38 14 PM" src="https://github.com/OllieWP/ollie/assets/1415737/4811ea85-5005-46d6-b4a0-f493545dee1f">
